presidio-analyzer/tests/conf/default.yaml

```diff
@@ -4,3 +4,16 @@ models:
     lang_code: en
     model_name: en_core_web_lg
 
+ner_model_configuration:
+  model_to_presidio_entity_mapping:
+    PER: PERSON
+    PERSON: PERSON
+    NORP: NRP
+    FAC: FACILITY
```

Comment:
Do we have a FACILITY entitiy in presidio?

Reply:
Not right now, and it hardly appears as an output of Spacy. One option would be to add it to the supported entities list, the other is to map it to location. WDYT?

Reply:
Map it to location IMO

Reply:
Done